### DISCUSSION TITLE
TV channels cutting out programs from their livestream

### DISCUSSION THREAD

**Discussion by kaikyoofers** [iptv-org/community#472] [Show and tell]
Seems like PBS Kids cut out Curious George from their livestreams, any thoughts about that?

Partial livestream: https://livestream.pbskids.org/out/v1/1e3d77b418ad4a819b3f4c80ac0373b5/est.m3u8

Full livestream: https://2-fss-2.streamhoster.com/pl_140/amlst:200914-1298290/playlist.m3u8

![Screenshot_20230810-123820_IPTV](https://github.com/iptv-org/iptv-org.github.io/assets/67206080/946e8eba-9d17-4466-bdda-f75e0fd61b96)

**Comment by benevenstanciano** [comment]
The first livestream is user-facing, and sometimes channels don't have rights to stream a TV show live.
The second livestream is to feed things like cable operators to air on TV, which they do have the rights to.
